master/usage/ipv6.md

```diff
@@ -1,31 +1,158 @@
 ---
-title: IPv6 Support
+title: Enabling IPv6 Support
 canonical_url: 'https://docs.projectcalico.org/v3.0/usage/ipv6'
 ---
 
-{{site.prodname}} supports connectivity over IPv6, between compute hosts, and
-between compute hosts and their VMs. This means that, subject to
-security configuration, a VM can initiate an IPv6 connection to another
-VM, or to an IPv6 destination outside the data center; and that a VM can
-terminate an IPv6 connection from outside.
-
-## Requirements for containers
-
-Containers have no specific requirements for utilizing IPv6
-connectivity.
-
-## Requirements for guest VM images
+### About IPv6 Support
+
+You may be interested in this document if you want to enable IPv6
+Support. This would be because:
```

Comment:
Please consider consolidating `### About IPv6 Support` and `### Purpose of this page`. This could be compressed into a single introduction section with links to get people directly to the section that they care about as follows:

```
## About enabling IPv6

After enabling IPv6:
- Workloads can communicate over IPv6.
- Workloads can initiate connections to IPv6 services.
- Workloads can terminate incoming IPv6 connections.

Support for IPv6 and the procedure for enabling it varies by orchestrator. Refer to the section that corresponds to your orchestrator for details.

- [Enabling IPv6 with Kubernetes](#enabling-ipv6-with-kubernetes)
- [Enabling IPv6 with OpenStack](#enabling-ipv6-with-openstack)
```